### PR TITLE
APIMANAGER-5474 : Integration tests

### PR DESCRIPTION
**Pull request by manoramahp**
Related fix : https://github.com/wso2/wso2-synapse/pull/673

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_APIM5474.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        </filter>
+        <filter xpath="$ctx:CUSTOM_HTTP_SC">
+            <then>
+                <property name="HTTP_SC" expression="$ctx:CUSTOM_HTTP_SC" scope="axis2"/>
+            </then>
+            <else>
+                <property name="HTTP_SC" value="500" scope="axis2"/>
+            </else>
+        </filter>
+        <class name="org.wso2.carbon.apimgt.usage.publisher.APIMgtFaultHandler"/>
```
Comment (praminda) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_APIM5474.xml]: We moved handlers to gateway component. So this should be `org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtFaultHandler`
Please update.

Comment (bhathiya) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_APIM5474.xml]: @manoramahp Please fix this.

Comment (manoramahp) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_APIM5474.xml]: Done

**Comment by bhathiya** [comment]
Conflicting files
modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

**Comment by manoramahp** [comment]
Closing since a new PR opened for 2.1.x branch
https://github.com/wso2/product-apim/pull/1733

**Pull request closed without merging**